# Q1: Write a Python Program to display a user enetered name 
# #followed by "Good afternoon" using Input function
# name = input("Enter YOur Name\n")
# print("Good Morning, "+ name)

# # Q2: write a python program to fill in a letter template given below 
# #with name and date
# letter = '''Dear,   <!name!>
#             you are selected
#             <!date!>'''
# name = input("enter your name\n")
# date = input("date\n")
# letter = letter.replace("<!name!>", name)
# letter = letter.replace( "<!date!>", date)
# print(letter)
# Q3 write a program todetect double spaces
# st = "hammad  is  a  good  Boy"
# doublespaces = st.find("  ") 
# print(st)
# print(doublespaces)
# print(st.replace("  ", " "))